[Merge on April 11] Remove Blackfire.io documentation from Cloud Guide (#6963)
magento/devdocs: src/cloud/live/live.md

*  Extensions may need to be installed and configured in a specific order. Some extensions touch similar pieces of code in the back and frontend, sometimes clashing and causing errors.
*  Extensions typically are heavily tested in [Developer mode]({{ site.baseurl }}/guides/v2.3/config-guide/bootstrap/magento-modes.html), not Production mode. When you move to Staging and use Production mode, you may find error and surprises between extensions, your custom code, and Magento.
*  Configurations and code may work great in your developer Integration environments and run into issues in Staging and Production. Working in your local does not confirm it works perfectly in production.

> *  Integration environments do not have all the incredible services available in Staging and Production, like Fastly and New Relic.

Author: ```suggestion
*  Integration environments do not support some services that are available in Staging and Production, like Fastly and New Relic.
```
